### ISSUE TITLE
Dashboards for non-admin users are corrupted (graphs / numbers not working)

### ISSUE THREAD

**Issue by ttytm** [invoiceninja/ui#1959]
![Screenshot_20240807_014253](https://github.com/user-attachments/assets/c86ff2fc-0244-46af-a161-1bd1b1984b4d)

Repro:
- Ensure there is an entry whose data should be displayed on the dashboard.
- Verify that it is visible as an admin.
- Try to display it with a non-admin user.

**Comment by turbo124** [comment]
If a user is not an administrator, they will only see the data of the records that they create.

**Comment by ttytm** [comment]
Thanks for the swift response @turbo124 .

This means that with this UI implementation it is not possible for a user who is not an administrator to get an overview of all invoices on the dashboard, but it possible to create reports for all of them?

This seems to be an inconsistent design decision. Is there a rationale for this?

**Comment by turbo124** [comment]
@ttytm we added the Reports/Dashboard permission levels after the view/edit/create permission levels based on feedback from users.

Whilst you make a valid point about the raw data still being accessible if reports are enabled the dashboard remains out of view based on some users preference.

**Comment by ttytm** [comment]
Is there a chance to add a permission option that would allow viewing the data on the dashboard when it is set?

**Comment by turbo124** [comment]
It can be a feature request, but in this scenario , most people would just use the administrator permission.

**Comment by ttytm** [comment]
Yes i think so too, it'll be the usual case. It was just a case I encountered where it would be more appropriate without promoting to admin rights. Thanks again.

**Comment by ttytm** [comment]
@turbo124, I hope I’m not bothering you too much. Would this be considered a feature request for the main repo then?

Since there is quite a gap between allowing a user to view full graphs vs promoting them to an admin, which includes permissions to create and delete organizations, it might not be suitable for some cases. I believe it would be appropriate to have a separate permission option for graph/stats access.

**Comment by turbo124** [comment]
@ttytm sure, i've set this as a feature request.
